oommfc/oommf/oommf.py

```diff
@@ -116,8 +111,9 @@ def errors(self):
         """
         pass  # pragma: no cover
 
+    @functools.cached_property
```

Comment:
- Regarding point 1: I don't think that this could happen. `platform` is part of `OOMMFRunner`, not the `Runner` class. So when changing the runner a new `platform` object (i.e. new `id`) should be called (and cached). However I did not test this - because I only have a single OOMMF installation (and no docker). Maybe you can try it? Knowing if it works as expected would be very interesting independent of whether we actually keep it or not. Would also be the option to `status` to check if OOMMF works (missing the version info).
- Regarding point 2: That's absolutely correct and I would be happy to change it to not being cached.

Reply:
Regarding point 1: you are probably right.

However, if you don't mind, then let us take the caching out here to keep it simple: the benefits are limited, and we may attract more worries in the long run.